Library/Homebrew/sorbet/rbi/homebrew.rbi

```diff
@@ -0,0 +1,19 @@
+# typed: strict
+
```

Comment:
This RBI file will be taking care of all method signatures from now on. `cmd/home.rb` is back to it's original state and I am making efforts to make sure that we can resolve all errors from this file itself without having to restructure any other file.

We're getting 7 new type errors now (13 errors => 20 errors), I'm working on resolving them now.

Reply:
Very nice. Yup, this looks perfect so far!